astro/src/content/blog/authn-authz-deployment-models-cerbos-webinar.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2025-04-24
+title: How To Choose The Right AuthZ And AuthN Deployment Model
+description: Self-hosted, single-tenant cloud, or multi-tenant SaaS. What is the best option for you?
+authors: Dan Moore
+image: /img/blogs/authn-authz-deployment-models-cerbos-webinar/how-to-choose-the-right-authz-and-authn-deployment-model.png
+categories: Education
+tags: webinar, ciam, authz, authn, modern applications, self-hosted, video, cerbos
+excerpt_separator: "{/* more */}"
+---
+
+Last week, FusionAuth and Cerbos co-hosted a webinar featuring myself and Alex Olivier, Cerbos CPO and co-founder. We shared insights on deploying both authentication (AuthN) and authorization (AuthZ) solutions, whether self-hosted, single-tenant in the cloud, or SaaS. 
+
+{/* more */} 
+
+We talked about how folks evaluating such solutions should go about finding the best fit for security, control, and scalability.
+
+## Understanding The Spectrum Of Deployment Options
+
+When deploying any architectural component that your application depends on, whether authentication, authorization or another type, you're making a trade-off between control and responsibility. 
+
+As I put it during our discussion, referring to the table below: "As you move from self-hosted to a multi-tenant fully managed SaaS, you have less and less responsibility but you have commensurately less and less control."
```

Comment:
"commensurately" requires a high reading level and isn't friendly to ESL readers. It sounds like this is a direct quote, but it would be more user-friendly to just remove the word from the sentence, as "less and less responsibility -> less and less control" works perfectly well.